index.bs

```diff
@@ -657,7 +671,9 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
       <p>
         The <dfn for='AudioConfiguration' dict-member>samplerate</dfn>
         represents the samplerate of the audio track in. The samplerate is the
-        number of samples of audio carried per second.
+        number of samples of audio carried per second. samplerate is only
```

Comment:
Samplerate is also applicable to file and media-source (though it's not really useful IMO, it's grandfathered)

Reply:
Fixed.